x-pack/legacy/plugins/uptime/public/components/functional/location_map/embeddables/embedded_map.tsx

```diff
@@ -45,29 +46,42 @@ const EmbeddedPanel = styled.div`
 `;
 
 export const EmbeddedMap = ({ upPoints, downPoints }: EmbeddedMapProps) => {
+  const { colors } = useContext(UptimeSettingsContext);
   const [embeddable, setEmbeddable] = useState<MapEmbeddable>();
-  const embeddableRoot: React.RefObject<HTMLDivElement> = React.createRef();
+  const embeddableRoot: React.RefObject<HTMLDivElement> = useRef<HTMLDivElement>(null);
   const factory = start.getEmbeddableFactory(MAP_SAVED_OBJECT_TYPE);
 
   const input = {
     id: uuid.v4(),
     filters: [],
     hidePanelTitles: true,
-    query: { query: '', language: 'kuery' },
-    refreshConfig: { value: 0, pause: false },
+    query: {
+      query: '',
+      language: 'kuery',
+    },
+    refreshConfig: {
+      value: 0,
+      pause: false,
+    },
     viewMode: 'view',
     isLayerTOCOpen: false,
     hideFilterActions: true,
-    mapCenter: { lon: 11, lat: 47, zoom: 0 },
+    mapCenter: {
+      lon: 11,
+      lat: 20,
```

Comment:
This section could use a comment explaining how these coordinates were chosen and what the goal is. I assume its to make the map more visually appealing (is this to crop the top / bottom)?

Reply:
Yes it's to crop top/bottom.